packages/components/src/ExampleComponent/ExampleComponent.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react"
```

Comment:
Even if we disregard the structure, the contents of your templates need a review (check or use the plop templates)😉